### DISCUSSION TITLE
Carbon Black Blocked Prisma Installaion

### DISCUSSION THREAD

**Discussion by stats** [prisma/prisma#14717] [Q&A] [answered]
I got the following error when installing Prisma through t3-create-app, windows 10.

The npm install failed on the file node_modules\@prisma\engines\prisma-fmt-windows.exe is stated that "APC is a known malware and was blocked".

I could not find reference to this issue anywhere else.

Prisma has previously installed without issue.

**Comment by austincrim** [comment, marked as answer]
Hey @stats 👋,

Is Carbon Black some kind of anti-virus software? What version are you running?

Do you get this error when installing Prisma? What version of Prisma?

**Reply by stats** [reply]
Carbon Black Cloud Version 3.8.0.398 is an endpoint security application from VMWare, this is a type of anti-virus/anti-malware solution.

I pulled down the latest version of create-t3-app which is trying to install @prisma/client ^4.2.0 and prisma ^4.2.0

The error occurs when allowing the create-t3-app to automatically run `npm install`

**Reply by janpio** [reply]
Thanks. 

We unfortuantely have trouble with one anti virus vendor (Cynet) detecting some of our engine files as malicious, but fortunately it does not seem to be popular enough to be triggered very often. You can see another case of this with the file you are reporting: https://www.virustotal.com/gui/file/9a300857c23a7447d16e6c39b7030bb04c87b3cf5f13833288e702c911b2cccd?nocache=1 But you can also see how all other Anti Virus vendors confirm that the file is totally fine.

I attached a zip file of the current `node_modules@prisma\engines\prisma-fmt-windows.exe` to this message. Can you download this and confirm this also triggers the detection of your anti virus as well? That would allow us to deduct that the real, unchanged version of the file is triggering on your system.

[prisma-fmt-windows.zip](https://github.com/prisma/prisma/files/9294938/prisma-fmt-windows.zip)

Does Carbon Black give you any additional information besides blocking the download?
Can you report false positives to Carbon Black? That would be very helpful.

**Reply by Jolg42** [reply]
I found out this page on their website:
https://community.carbonblack.com/t5/Knowledge-Base/All-Products-How-to-report-Malware-False-Positives-to-VMware/ta-p/101821

Question
If hash is classified as malware, but it should be trusted approved, how should this be reported to VMware Carbon Black?

Answer
Please [Open a Support case](https://community.carbonblack.com/t5/Groups/ct-p/user-groups?case_Page=case_create) with the following information:
- The affected hash(es)
- Current Reputation
- Expected Reputation
- Associated Events
- Alert ID (if available)
- Reason(s) why the hash is believed to be legitimate

@stats Could you open a support case with the information needed maybe? 
Also, if you could share with us the requested information, it would be interesting.

**Reply by stats** [reply]
Yes, the zip file version also resulted in the same threat detection and block.

I have reported this to our IT Department as a false positive and requested they enter a ticket with VMWare as a false positive.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
